boards/common/esp32/Makefile.include

```diff
@@ -8,4 +8,10 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 include $(RIOTMAKE)/tools/serial.inc.mk
 
 # reset tool configuration
-export RESET = esptool.py --before default_reset run
+ifeq (1,$(ESP_RESET_DTRTOGGLE))
+  # use dtr tool from https://github.com/kaspar030/rtsdtr for reset
```

Comment:
Is this something you have written especially for ESP32 or is it used by other boards too?

Reply:
I wrote that specifically for ESP32. I think it can be used for other boards that use a combination of rts and dtr for reset, e.g., msba2 or waspmote-pro.
See below for why we need this (or similar).